lib/common/schemas.c

```diff
@@ -1222,7 +1236,7 @@ pcmk_update_configured_schema(xmlNode **xml, bool to_logs)
 
         entry = NULL;
         converted = pcmk__xml_copy(NULL, *xml);
-        if (pcmk__update_schema(&converted, NULL, true, to_logs) == pcmk_rc_ok) {
+        if (pcmk__update_schema(&converted, NULL, true, out) == pcmk_rc_ok) {
```

Comment:
We can just drop the to_logs argument from `pcmk__update_schema()` (and the functions further down the call chain)